POEApi.Model/LegionType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace POEApi.Model
+{
+    public enum LegionType
```

Comment:
Let's call this `LegionFaction`, so it's clear what is being referred to.

Reply:
Done - rename additional symbols